File: CodeForces/734A. Anton and Danik/Anton_and_Danik.cpp
// Solution By: Mahmoud Akram
#include <iostream>

using namespace std;

int main()
{
    int anton=0,danik=0,n;
    string games;
    cin>>n>>games;
    for (auto win:games){
        if(win=='A')
            anton++;
        else
            danik++;
    }
    if(anton>danik)
        cout<<"Anton";
    else if(anton==danik)
        cout<<"Friendship";
    else
        cout<<"Danik";

}

File: CodeForces/734A. Anton and Danik/Anton_and_Danik.py
# Solution by : Yousef Nefea 
a = 0
d = 0
_ = input()
inp = input()
for i in inp:
    if i == "A":
        a+=1
    else:
        d+=1
if a == d:
    print("Friendship")
elif a > d:
    print("Anton")
else:
    print("Danik")
        